comCam/manual_defects/r22_s00/20241120T000000.ecsv

```diff
@@ -0,0 +1,33 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - name: x0
+#   unit: pix
+#   datatype: int32
+#   description: X coordinate of bottom left corner of box
+# - name: y0
+#   unit: pix
+#   datatype: int32
+#   description: Y coordinate of bottom left corner of box
+# - name: width
+#   unit: pix
+#   datatype: int32
+#   description: X extent of box
+# - name: height
+#   unit: pix
+#   datatype: int32
+#   description: Y extent of box
+# meta: !!omap
+# - OBSTYPE: defects
+# - INSTRUME: LSSTComCam
+# - DETECTOR: 1
+# - CALIBDATE: '2024-11-19T09:00:00'
+# - CALIB_ID: raftName=R22 detectorName=R22_S00 detector=0
+# - DEFECTS_SCHEMA: Simple
+# - DEFECTS_SCHEMA_VERSION: 1
+# - DATE: '2024-11-19T09:00:00'
+# - CALIB_CREATION_DATE: '2024-11-19'
+# - CALIB_CREATION_TIME: 09:00:00
+# schema: astropy-2.0
+x0 y0 width height
+# Empty defect
```

Comment:
I think we want to fix these github "missing newline at end" errors, just to avoid potential problems in the future.